e2e_tests/tests/trial/support-bundle.py

```diff
@@ -0,0 +1,24 @@
+import subprocess
```

Comment:
meta comment, I'd put this in `e2e_tests/tests/cluster/test_support_bundle.py` instead